hack/generate/cli-doc/gen-cli-doc.go

```diff
@@ -14,16 +15,168 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// This program is a hacked version of the cobra cli-doc generator to generate
```

Comment:
Please put this in `hack/third_party/github.com/spf13/cobra`. Thanks!

Reply:
Would also be nice to split into 2 commits:
1. Copy in the cobra file unmodified
2. Modify the file

Reply:
Noted